apps/trading-e2e/src/support/step_definitions/deposits.step.ts

```diff
@@ -0,0 +1,77 @@
+import { And, Then, When } from 'cypress-cucumber-preprocessor/steps';
+import { EthereumWallet } from '../ethereum-wallet';
+import DepositsPage from '../pages/deposits-page';
+
+const depositsPage = new DepositsPage();
+const ethWallet = new EthereumWallet();
+
+const tBTC = '5cfa87844724df6069b94e4c8a6f03af21907d7bc251593d08e4251043ee9f7c';
+const invalidPublicKey =
+  'zzz85edfa7ffdb6ed996ca912e9258998e47bf3515c885cf3c63fb56b15de36f';
+const validPublicKey =
+  'f8885edfa7ffdb6ed996ca912e9258998e47bf3515c885cf3c63fb56b15de36f';
```

Comment:
Lint error and also this could be using the Cypress env stored under `vegaPublicKey`